### PR TITLE
move the  branch to pensa-repo

### PR DESCRIPTION
**Pull request by MajidTaheri**
**Related PR**

see https://github.com/Pensasystems/pensa-ros/pull/965

**Describe the solution**
`t265realsense_camera` will Publish `slam_error` and `Stream transfer failed, exiting` to ` /diagnostics` topic. so in pensa_ros we have this ability to subscribe to this topic and capture the T265 errors.

### CODE REVIEW

**Review by adam-pensa — APPROVED** [review]
LGTM. I had never seen `std::find_if`, pretty cool! This is some nice modern C++

File: realsense2_camera/src/t265_realsense_node.cpp [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #include "../include/t265_realsense_node.h"
+#include <functional>
```
Comment (marcelino-pensa) [inline realsense2_camera/src/t265_realsense_node.cpp:2]: what is the purpose of this library?

File: realsense2_camera/src/t265_realsense_node.cpp [hunk 2]
```diff
@@ -117,3 +135,9 @@ void T265RealsenseNode::calcAndPublishStaticTransform(const stream_index_pair& s
         }
     }
 }
+
+void T265RealsenseNode::warningDiagnostic(diagnostic_updater::DiagnosticStatusWrapper& status)
+{
+  status.summary(diagnostic_msgs::DiagnosticStatus::WARN, _T265_fault);
+
```
Comment (marcelino-pensa) [inline realsense2_camera/src/t265_realsense_node.cpp:142]: delete this extra line here

**Review by marcelino-pensa — APPROVED** [review]
This looks great to me! Next step would be to also merge this into Intel and leave the ball on their court

**Pull request merged**